docs/ref/release_notes.rst

```diff
@@ -0,0 +1,54 @@
+What's new in Iris-grib v0.13
+=============================
+
+:Release: 0.13.0
+:Date: T.B.D.
+
+Features
+--------
+
+* Added loading for data on Hybrid Height levels (surface type 118 in
+  code table 4.5).
+
+Bug Fixes
+---------
+
+* Fixed a bug with loading data on Hybrid Pressure levels (surface types 105
+  and 119 in code table 4.5).  
+  Previously, *all* hybrid coordinate values, in both 'level_pressure' and
+  'sigma' coordinates, were loaded from the next level up,
+  i.e. (model_level_number + 1).
+
+  .. note::
+
+      This changes loading behaviour : previous results were simply wrong.
```

Comment:
Maybe we should use a slightly different way of wording this than 'simply wrong' (which is true, but a little blunt).  How about 'incorrect'?

Reply:
The point I'm trying to make is that, although this a breaking behaviour change, we can allow that because the existing behaviour is definitely "wrong", i.e. you can't really defend what it currently does as being of use to anyone (!)
I'll try + come up with something milder ..

Reply:
@pp-mo I support the suggestion of @corinnebosley that we could do with some milder wording for this!

We could also make it absolutely clear that the changed load behaviour affects _only_ loading hybrid pressure levels.